public/react-services/wz-request.ts

```diff
@@ -156,4 +152,18 @@ export class WzRequest {
         : Promise.reject(error.message || error);
     }
   }
-}
\ No newline at end of file
+
+  /**
+   * Customize message and return an error object
+   * @param error 
+   * @param message 
+   * @returns error
+   */
+  static customErroMessage(error, message){
```

Comment:
**issue**: if the method is `static` to the class, this is not accessible by the class instance.

**suggestion**: Remove the `static` keyword of the method or replace `this` with the class name when using it.

Explanation:
```
// Define a class
class MyClass{
  // Constructor of the class instances
  constructor(){
    // Initialize the class instance. `this` keyword does reference to the class instance
  }
  // Define a static method for the class
  static staticMethod(){
    // This method is static of the class. This can't be accesed by the class instance.
    // Use:
    // MyClass.staticMethod();
  }
  
  // Define a method accesible by the class instance throught the prototype chain
  methodAccesibleByTheClassInstance(){
    // This method is in the prototype chain of the class instance, so this is accesible.
    // Use:
    // this.methodAccesibleByTheClassInstance();
  }
}
```

**nitpick**: The method name has a typo `Erro` instead of `Error`.